lib/index.js

```diff
@@ -42,15 +44,19 @@ const Rooftop = {
         // params: { per_page: 999999 }
       }).wrap(pathPrefix, {
         prefix: url
-      })
+      }).wrap(template, { params: {} })
 
     return new Proxy({
       name: opts.name,
       apiToken: opts.apiToken,
       contentType: (name) => {
         return {
           get: (opts) => {
-            return client(Object.assign({ path: name }, { params: opts }))
+            let pathTemplate = name
+            if (opts) {
+              for (let param in opts) { pathTemplate += `{?${param}}` }
+            }
```

Comment:
[objectfn](https://github.com/jescalan/objectfn) could help to abbreviate this if you want!